filebeat/etc/beat.yml

```diff
@@ -151,6 +151,9 @@ filebeat:
   # Event count spool threshold - forces network flush if exceeded
   #spool_size: 2048
 
+  # Enable async publisher pipeline in filebeat (Experimental!)
```

Comment:
As it is experimental, I would probably not put it in the config here. Yes I know, somehow breaks our rule so far.